truss/templates/control/requirements.txt

```diff
@@ -6,7 +6,7 @@ loguru>=0.7.2
 python-json-logger>=2.0.2
 tenacity>=8.1.0
  # To avoid divergence, this should follow the latest release.
-truss==0.9.96
+truss==0.9.100
```

Comment:
This is probably going to break unit / integration tests until this version of truss is actually released, might be worth to do anyways